std_msgs/Header header
float32 height
float32 width
float32 inner_radius